test/kafka.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    version: 2.1.0
+    replicas: 1
+    listeners:
+      plain: {}
```

Comment:
Are those empty nodes required?

Reply:
Theses are a straight copy from the strimzi installation instructions - aiming to keep things straightforward, so periodically we can update to more recent versions.